cmd/minikube/cmd/node_add.go

```diff
@@ -36,13 +41,46 @@ var (
 	cpNode              bool
 	workerNode          bool
 	deleteNodeOnFailure bool
+	osType              string
+	// windowsVersion      string
+
+	osTypeLong = templates.LongDesc(i18n.T(`
```

Comment:
Why do you need `templates.LongDesc`? Also minikube has a built-in translation tool, so you shouldn't need `i18n.T`.

Reply:
Removed this package!